File: ward_connect/server/models/Scheme.js
const mongoose = require("mongoose");

const schemeSchema = mongoose.Schema({
  sid: {
    required: true,
    type: String,
    trim: true,
  },
  sdetails: {
    required: true,
    type: String,
    trim:true
  },
  stype:{
    required:true,
    type:String,
    trim:true
  }
});

const scheme = mongoose.model("schemes", schemeSchema);
module.exports = scheme;

File: ward_connect/lib/models/scheme.dart
import 'dart:convert';

class Scheme {
  final String sid;
  final String sdetails;
  final String stype;

  Scheme({
    required this.sid,
    required this.sdetails,
    required this.stype,
  });

  factory Scheme.fromJson(Map<String, dynamic> json) {
    return Scheme(
      sid: json['sid'],
      sdetails: json['sdetails'],
      stype: json['stype'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'sid': sid,
      'sdetails': sdetails,
      'stype': stype,
    };
  }
}
